db/schema.rb

```diff
@@ -14,6 +14,7 @@
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
+  enable_extension "hstore"
```

Comment:
Was this something that you added in yourself @wanyamaman? or was it generated somehow? 😄